types/cli-interact/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for cli-interact 0.1
+// Project: https://github.com/zhami/cli-interact
+// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { BasicOptions } from 'readline-sync';
+
+export interface ChoiceOptions {
+    allowNoAnswer?: boolean;
+    returnNumeric?: boolean;
+}
+
+export interface NumberOptions {
+    allowNoAnswer?: boolean;
+    requireInteger?: boolean;
+}
+
+export function getChar(promptText: string, allowedCharsAsString: string, flagAllowNoAnswer?: boolean): string;
+export function getChoice(title: string, choices: string[], opts: NumberOptions & {returnNumeric: true}): number;
+export function getChoice(title: string, choices: string[], opts?: ChoiceOptions): string;
+export function getChoiceByChar(title: string, choices: string[], flagAllowNoAnswer?: boolean): string;
+export function getInteger(promptText: string): number;
+export function getIPversion(flagAllowNoAnswer?: boolean): string;
+export function getNumber(promptText: string, opts?: boolean | NumberOptions): number;
```

Comment:
What does a `Boolean` indicate?  Whether an integer is required?

Reply:
Yes. In the source code it says "for historical compatibility".